enterprise/server/remote_execution/containers/firecracker/test.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+
+# Firecracker tests have to be run as root since they use FUSE mounts.
```

Comment:
This script probably seems overkill, but the alternatives kind of suck -

* `bazel run <target> --run_under=sudo` didn't seem to work, the test just hangs for some reason
* `bazel test <target> --run_under=sudo` doesn't work because the input is not a terminal and it needs to ask for the root pass. `SUDO_ASKPASS` helper is an option but seems hard to implement in a secure way.
* `sudo bazel` results in weird "permission denied" errors when trying to read source files, also kind of sucks because it uses a totally separate Bazel cache namespace